scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -157,47 +159,47 @@ protected static void start(CommandLine commandLine) throws Exception {
         hsqldbServer = new SchedulerHsqldbStarter();
         hsqldbServer.startIfNeeded();
 
-        String rmUrl = getRmUrl(commandLine);
+        rmURL = getRmUrl(commandLine);
+        //        rmURL = rmUrl;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)